### PR TITLE
feat(video-player): introduces react wrapper

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#8657 

### Description

Introduces `video-player` React wrapper.

### Changelog

**New**

- `video-player` react wrapper

### CODE REVIEW

**Comment by RichKummer** [comment]
@kennylam Percy found some cases where the `ToC - vertical` was shifted back from where Ari's update bumped it. Does this need to be merged down so it doesn't overwrite?

Screen Shot 2022-06-16 at 6 06 28 PM

**Comment by IgnacioBecerra** [comment]
@kennylam Just missing the CodeSandbox example for the React wrapper, but looks good! Glad the wrapper didn't end up requiring more work to get it going. 😌

**Comment by kennylam** [comment]
> Percy found some cases where the `ToC - vertical` was shifted back from where Ari's update bumped it. Does this need to be merged down so it doesn't overwrite?

@RichKummer Pretty sure this is a false positive. Nothing in this PR would cause these changes.

**Comment by kennylam** [comment]
> Just missing the CodeSandbox example...

@IgnacioBecerra Doh, knew I forgot something. Good catch!

**Comment by annawen1** [comment]
@kennylam looking good, looks like you just need to do a `yarn format`

**Comment by kennylam** [comment]
@annawen1 Good catch, thanks!

File: packages/web-components/src/components/video-player/__stories__/README.stories.react.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import { PropTypesRef } from '@carbon/ibmdotcom-web-components/es/components-react/video-player/video-player';
+
+# Tag Group
```
Comment (annawen1) [inline packages/web-components/src/components/video-player/__stories__/README.stories.react.mdx:5]:
```suggestion
# Video Player
```

**Review by annawen1 — APPROVED** [review]
Looks good! just that one edit to the docs

**Pull request merged**